src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -1632,11 +1630,11 @@ static void UCS_F_ALWAYS_INLINE uct_rc_mlx5_iface_copy_iov_to_dm(
         log_sge->num_sge = 0;
     }
 
-    while (uct_iov_to_buffer(iov, iovcnt, offset, &word, sizeof(word)) != 0) {
+    while (iov_iter.iov_index != iovcnt) {
+        word = 0;
+        uct_iov_to_buffer(iov, iovcnt, &iov_iter, &word, sizeof(word));
```

Comment:
we need to fetch multiple words at once: ucs_round_down_pow2(iov length to copy, sizeof(word))

Reply:
@yosefe updated this part to copy multiple words from iov->buffer to device memory.
Could you please to review the changes.